support-frontend/app/controllers/Application.scala

```diff
@@ -7,7 +7,7 @@ import cats.data.EitherT
 import cats.implicits._
 import com.gu.i18n.CountryGroup
 import com.gu.i18n.CountryGroup._
-import com.gu.identity.play.IdUser
```

Comment:
previously identity-play-auth was shipping it's own models for identity data, but if these are getting imported then identity-model should be used!